---
layout: page
title: Research
---

I am generally interesting in data analysis and modeling. I have already plunged into analyses of human genetic data, (raw) deep sequencing data but and, of course, simulated datasets. I have performed genome-wide association analyses (linear/logistic regression and dimensionality reduction methods), mixed-effect modelling but also studied stochastic birth-death models and coalescent models. Currently, I focus on phylodynamic modeling (birth-death and coalescent models) and methods development.

**Current project - method development**
Phylogenetic tools applied to sequence data reconstruct a phylogenetic
tree, a network structure that displays how the individuals in the
population are related to each other. Phylodynamic tools take the
inference a step further, by providing an estimate of the dynamics of
the population that these individuals came from. When set in a Bayesian
statistics framework, inferences of the phylogeny and the population
dynamics are done simultaneously, rather than in a step-wise manner as
in the maximum likelihood approach. This allows for possible uncertainty
in the inference of the phylogeny to be integrated over when inferring
the population dynamic parameters, but it also poses higher computational
burden. Thus, usefulness of Bayesian phylodynamics for large datasets is
still limited due to the computational effort required to process more
than a few hundred sequences. I develop fast methods to advance the use
of Bayesian phylodynamics for thousands of sequences that usually come out from deep sequencing projects.
